### ISSUE TITLE
Azure Storage Explorer connection error

### ISSUE THREAD

**Issue by kturngan** [microsoft/AzureStorageExplorer#3399]
Hi, 

I am getting the error below while others in my organization have managed to access the File Shares using the same connection string:

Unable to retrieve child resources.

Details:
The provided account key <deleted> is not a valid base64 string.

Apparently it is also supposed to prompt for a password but I am not being prompted.

Thanks
Karl

**Comment by MRayermannMSFT** [comment]
@kturngan how are you accessing your resources? It looks like you are probably using name and key? If so, please make sure your key is correct, or check if they key for the account has been rotated. If it is wrong or if they key has been rotated, you'll need to re-attach using the correct value.

**Comment by kturngan** [comment]
Thanks @MRayermannMSFT I am accessing via a connection string provided to me. Should I just contact the administrators and ask for a new key? If I do this, would it affect everyone else who currently have access?

**Comment by MRayermannMSFT** [comment]
Yes, I would ask your admins to confirm the value of the account key.

If the key has been rotated, that would affect everyone who uses it. If it's merely a typo, then that would only affect you.

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. If you are still having issues please open a new issue.
